time stamp
float32 left
float32 right
